derive/src/serde_json.rs

```diff
@@ -95,14 +95,14 @@ pub fn derive_de_json_named(name: &str, defaults: bool, fields: &[Field]) -> Tok
                     val = format!("\"{}\".to_string()", val)
                 }
                 if field.ty.is_option {
-                    val = format!("Some({})", val);
+                    val = format!("::std::option::Option::Some({})", val);
                 }
                 Some(val)
             } else {
                 if !field.ty.is_option {
-                    Some(String::from("Default::default()"))
+                    Some(String::from("::std::default::Default::default()"))
```

Comment:
afaik Option/Into are pretty safe to use without `::`

Reply:
also `core::option` is the better choice to get an `Option` from. But is it really a necessity to use `::` for an Option?

Reply:
True, but there's no downside, and it's generally recommended to use fully qualified paths everywhere in macros, just in case the user happens to be completely mad.

Reply:
any links on a scenario when an Option is not available?

Reply:
Not links, but the user *could* be operating in an environment without the standard library's prelude, but with the standard library available (not sure how you'd do that, but I assume you could). What are your reservations?

Reply:
yes, but both Option and Into are parts of `core`, not just `std`

Reply:
Oh yes of course, sorry. I don't think that changes anything though, as there are no downsides to having fully-qualified paths for these. I can remove them if you'd like, but I don't see any reason; better safe than sorry, right?